### ISSUE TITLE
Ubuntu hanging after kernal panic

### ISSUE THREAD

**Issue by nicholasrobertm** [zimmertr/Bootstrap-Kubernetes-with-QEMU#9]
Rolling this with an ubuntu image (of any kind) causes it to hang after kernal panic. this looks to be related to what's in this:
 https://kb.vmware.com/s/article/52683
Thread.

Basically, if you don't start the VM with a serial port, it will get stuck in kernal panic. Solving this can be done with qm set vmid --serial 0 or via doing it in the UI during the process before the reboot happens

**Comment by zimmertr** [comment]
Ohhhhh Hey. Nice find! Choosing a base image was a bit troublesome. CentOS is my preferred distro but they have hardcoded some nameservers which make it a broken option. I opened a bug report with them months ago but they haven't interacted with it aside from verifying the existence. -_- 

https://bugs.centos.org/view.php?id=15426

I ended up going with Debian as a second choice but, like you mentioned, it kernel panics on the first boot. I addressed this with an incredibly hacky solution as seen here: https://github.com/zimmertr/Bootstrap-Kubernetes-with-QEMU/blob/master/playbooks/main/deploy_qcow2_vms.yml#L97

I'll review your reference document in due time and try and get a fix pushed up. It might be a little bit though, I'm admittedly facing some minor burnout right now and not actively developing anything. PRs are always welcome. :)
